projects/demo/src/modules/components/input-date-range/input-date-range.template.html

```diff
@@ -14,6 +17,46 @@ <h3 i18n>DI-tokens for date localization:</h3>
             <dt><code>TUI_DATE_SEPARATOR</code></dt>
             <dd>single-character date's separator (dot, slash etc.)</dd>
         </dl>
+        <p class="tui-space_bottom-9">
+            <a tuiLink routerLink="." fragment="date-localization">
+                See example
+            </a>
+            with the usage of these tokens.
+        </p>
+
+        <h3 i18n>DI-tokens for input-configurations:</h3>
+        <dl i18n>
+            <dt><code>TUI_DATE_RANGE_VALUE_TRANSFORMER</code></dt>
+            <dd>
+                custom format of control output (
+                <a
+                    tuiLink
+                    target="_blank"
+                    href="https://github.com/Tinkoff/taiga-ui/blob/main/projects/cdk/date-time/day-range.ts"
+                >
+                    TuiDayRange
+                </a>
+                is default).
+                <p>
+                    We recommend to do it using another token
```

Comment:
I'm not sure we recommend anything here, maybe just change it to something like `See an example of how to provide transformers for InputDate and InputDateRange to work with native Date objects`?

Reply:
fixed